src/components/Bullet.tsx

```diff
@@ -52,10 +60,26 @@ const Bullet = ({
       graypulse: pending,
       'show-contexts': showContexts,
       'invalid-option': invalid,
+      'bullet-highlighted': isDragging,
     })}
   >
-    <span className='glyph' onClick={onClick}>
-      {glyph || (showContexts ? (isLeaf ? '◦' : '▹') : isLeaf ? '•' : '▸')}
+    <span
+      className='glyph'
+      style={{ width: `${Math.round(fontSize / 2) * 2}px`, height: `${Math.round(fontSize / 2) * 2}px` }}
```

Comment:
Let's add a descriptive comment explaining the need for the rounding since it is not obvious from the code. You can include a link to the issue.